script/example.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+import yaml
+import json
+from argparse import ArgumentParser
+
+def action_json(args):
+
+    with open('scripts/example_data.json') as json_file:
```

Comment:
Consider using `json.load(json_file)` instead of `json.loads(json_file.read())` to simplify the code and potentially improve performance by avoiding reading the entire file into memory.
```suggestion
print(json.dumps(json.load(json_file), indent=2))
```